test/parallel/test-child-process-detached.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
-require('../common');
+var common = require('../common');
```

Comment:
Right, I kept with whatever was being used in the file, for least-confusion/impact.

I think `const`-ifying tests makes more sense in separate PR(s)/commit(s). SGTY?